src/sql/workbench/services/dialog/browser/wizardModal.ts

```diff
@@ -175,6 +176,7 @@ export class WizardModal extends Modal {
 
 	public async showPage(index: number, validate: boolean = true, focus: boolean = false, readHeader: boolean = true): Promise<void> {
 		let pageToShow = this._wizard.pages[index];
+		const prevPageIndex = this._wizard.currentPage;
```

Comment:
Is this going to be -1 when showing the first page?

Reply:
Nope! on the first page, it is coming as '0'.

Reply:
The previous page is? That doesn't seem correct - that would mean that the index and the prevPageIndex would both be 0?

If that's the case would you mind opening an issue about that and assigning it to me? It doesn't need to be fixed as part of this PR.

Reply:
Sure, will open an issue, Thanks :)